src/Resources/public/js/fragmentList.jquery.js

```diff
@@ -37,6 +37,10 @@
             this.build();
             this.setEvents();
             this.setDraggableElements();
+            var divElement = $('<div>');
+            divElement.attr('id', 'deletedFragments');
```

Comment:
think we should follow some kind of convention here ;) id should be hypens separated as mainly in sonata admin templates